package.json

```diff
@@ -14,7 +14,8 @@
     "lint:ci": "yarn run eslint --quiet --format junit -o reports/lint/junit-lint.xml",
     "prettier:check": "yarn prettier --check .",
     "prettier:write": "yarn prettier --write .",
-    "circleci:update-config": "{ cat .circleci/autogenerated_header.yml; circleci config pack .circleci/src; } > .circleci/config.yml && circleci config validate"
+    "circleci:update-config": "{ cat .circleci/autogenerated_header.yml; circleci config pack .circleci/src; } > .circleci/config.yml && circleci config validate",
+    "create-release-note": "bash ./release-notes/createReleaseNotes.sh $1"
```

Comment:
Not part of this ticket, just created a litte script to make release-note-creation easier.